src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -112,8 +116,12 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       return this.isOV() && this.options.appState.getSchemaByName('autoChallenge');
```

Comment:
since we are checking for the presence of `autoChallenge` in `this.isAutoChallengeMissingFromRemediation()` do we have to check again here? can it be just `return this.isOV();` and same for` isCustomAppWithAutoChallenge` to `return this.isCustomApp();`

Reply:
hi @indirathirumalaimurugan-okta , good point. The check for `autoChallenge` in the appState is now a bit redundant so i updated the PR and got rid of the `isCustomAppWithAutoChallenge` and `isOVWithAutoChallenge` methods.

Everything can now be deduced from `isOV`, `isCustomApp`, and `isAutoChallengeMissingFromRemediation`